examples/reliable_conversation/src/utils/log_formatter.py

```
@@ -0,0 +1,199 @@
+                formatted_msg['content'] = format_message_content(msg['content'])
+                formatted_messages.append(formatted_msg)
+            else:
+                formatted_messages.append(msg)
+        
+        formatted_data['messages'] = formatted_messages
+        return json.dumps(formatted_data, indent=2)
+    
+    # Handle other structured data
+    try:
+        return json.dumps(data, indent=2)
+    except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)